bot/cogs/help.py

```diff
@@ -110,9 +110,9 @@ def _get_query(self, query: str) -> Union[Command, Cog]:
         cog = self._bot.cogs.get(query)
         if cog:
             return Cog(
-                name=cog.__class__.__name__,
+                name=cog.cog_name,
                 description=inspect.getdoc(cog),
-                commands=[c for c in self._bot.commands if c.instance is cog]
+                commands=[c for c in self._bot.commands if c.cog_name == cog.cog_name]
```

Comment:
```suggestion
                commands=[c for c in self._bot.commands if c.cog_name == cog.qualified_name]
```

Reply:
More akin to what was there before

```suggestion
                commands=[c for c in self._bot.commands if c.cog is cog]
```

Reply:
Which of these two suggestions are intended?

Reply:
Either one. Sorry, I'll add some more context. The latter one would be nicer except that the `cog` attribute is undocumented. Supposedly it would still be considered a "breaking change" were it to be renamed or removed by discord.py.